.github/workflows/pr_presubmit.yaml

```
@@ -23,20 +23,34 @@ jobs:
     runs-on: ${{matrix.os}}
     steps:
     - uses: actions/checkout@v2
+
     - uses: actions/setup-go@v3
       with:
-        go-version: 1.x
+        go-version: 1.24.x
         cache: true
+
     - id: files
       uses: Ana06/get-changed-files@v2.2.0
-    - uses: actions/checkout@v2
+
+    - name: Install workflow tools
+      run: make -f tasks.mak makefile_symlink && make install_tools
```

This can be run in one command.
```suggestion
      run: make -f tasks.mak makefile_symlink install_tools
```